pycon/static/less/bootstrap/mixins.less

```diff
@@ -558,13 +558,13 @@
   .core (@gridColumnWidth, @gridGutterWidth) {
```

Comment:
These changes don't look like part of this pull request.  I'm not really sure what they are.